steps/appium-server/0.0.4/step.yml

```
@@ -0,0 +1,42 @@
+title: Run Appium Server
+summary: |
+  Install and Run Appium Server
+description: |
+  Install and Run Appium Server on Bitrise to be able to run our mobile automated tests
+website: https://github.com/moatazeldebsy/bitrise-step-appium-server
+source_code_url: https://github.com/moatazeldebsy/bitrise-step-appium-server
+support_url: https://github.com/moatazeldebsy/bitrise-step-appium-server/issues
+published_at: 2021-08-27T16:04:04.474263+02:00
+source:
+  git: https://github.com/moatazeldebsy/bitrise-step-appium-server.git
+  commit: 6a427500aee5a429f61d1bda9632a7e30ec455d0
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+type_tags:
+- utility
```

```suggestion
- test
```

I think test would be a better category for this step

---

I think because I'm installing the server only so I added it as `utility`